typescript/nomad-deploy/scripts/twoLocals/bridge.ts

```diff
@@ -1,5 +1,5 @@
 import { getPathToDeployConfig } from '../../src/verification/readDeployOutput';
-import { deployBridgesComplete } from '../../src/bridge';
+import { deployBridgesHubAndSpoke } from '../../src/bridge';
```

Comment:
rename folder "twoLocals" to "local"

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/17/commits/ff75273178b6c024dbce8b40399362d8e5c5d680